lib/src/command/lish.dart

```diff
@@ -126,12 +126,12 @@ class LishCommand extends PubCommand {
       if (error.statusCode == 401) {
         msg += '$host package repository requested authentication!\n'
             'You can provide credentials using:\n'
-            '    pub token add $host\n';
+            '    dart pub token add $host\n';
```

Comment:
Could you do
```suggestion
            '    $topLevelProgram pub token add $host\n';
```
to support `flutter pub token add`